Sources/Tracking/CustomerIO.swift

```diff
@@ -84,6 +97,14 @@ public extension CustomerIOInstance {
     ) {
         track(name: name, data: EmptyRequestBody(), jsonEncoder: jsonEncoder, onComplete: onComplete)
     }
+
+    func screen(
+        name: String,
+        jsonEncoder: JSONEncoder? = nil,
```

Comment:
No need to include the `jsonEncoder` param since there is not `body` param.

Reply:
👍